modules/movies_ui.js

```diff
@@ -0,0 +1,73 @@
+import configureListener from './configure_listeners.js';
+import { handleComment, handleLike } from './event_listeners.js';
+
+import { toJson } from './json_helper.js';
+import LikesService from './likes_service.js';
+// import Movie from './movie';
+import MovieService from './movie_service.js';
+
+export const topRatedSection = document.querySelector('.top-rated');
+const popularSection = document.querySelector('.popular');
+
+const createMovieCard = (movie) => {
+  const tmp = document.createElement('tmp');
+  // Check the data attribute you will use it to create the popup, retrieve it back with fromJson
+  tmp.innerHTML = `<article class="movie-container" data=${toJson(movie)}">
+    <div class="movie-card">
+    <img src="${movie.image}" alt="${movie.title}">
+    <div class="movie-data">
+      <p class="title"> ${movie.title} </p>
+      <div class="rate-wrapper">
+      <span class = "fa fa-star checked"></span>  
+      <p class="rate"> ${movie.rate} </p>
+
+    </div>
+    </div>
+
+    <div class="movie-interaction id=${movie.id}">
+  <article class="comment-btn" id=${movie.id}> <span class="fa-solid fa-comment" id=${movie.id}></span></article>
+      <article class="like-btn" ><span  id=${movie.id} class="fa-regular fa-heart"></span> <span class="likes-count">${LikesService.getOneItemLikes(movie.id)}</span></article>
+    </div>
+  </article>`;
+
+  /*
+  1- Each html element has query selector function so you can get any element inside it
+  2- you cann pass functions as arguments.
+  3- Ex: we have like-button element inside that temp.
+  4- we can get that button from the temp becauze it its parent
+  */
+  // configureListener(tmp, handleSubmit, { childClassName: '.button1' });
```

Comment:
- [ ] **Required:** Kindly remove all unused code from your codebase. It makes your work look more professional.

Reply:
I was explaining to my partner how this function works but yeah you are right ✨✌️